chore!: rename fee juice

alexghr opened the pull request:
Fix #7570

Commit:
chore: rename fee juice

alexghr commented on docs/docs/migration_notes.md:
I think maybe I should undo the renames in this file and just add new migration steps.

rahul-kothari replied:
yea!

rahul-kothari commented:
the protocol specs docs also needs updates i think - I saw the use of fee paying asset a lot

Commit:
chore: more renames

Commit:
chore: update protocol docs

Commit:
chore: update migration notes

alexghr commented:
Updated protocol docs and setup a redirect from the old page to the new `/fee-juice` one. Also restored the migration notes from before the rename and added a short description about the change under 0.48.0

Commit:
chore: more renames

Commit:
chore: update fee juice address
